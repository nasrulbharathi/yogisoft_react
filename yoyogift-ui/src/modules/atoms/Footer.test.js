import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";
import renderer from "react-test-renderer";

describe("Footer", () => {
  let mountedFooter;
  beforeEach(() => {
    mountedFooter = shallow(<Footer />);
  });
  it("should be defined", () => {
    expect(Footer).toBeDefined();
  });
  it("should render Footer", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
