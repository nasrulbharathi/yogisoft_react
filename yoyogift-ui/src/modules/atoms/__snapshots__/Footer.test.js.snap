// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer 1`] = `
<div
  className="Footer-root-1"
>
  <header
    className="MuiPaper-root-11 MuiPaper-elevation4-17 MuiAppBar-root-2 MuiAppBar-positionStatic-6 MuiAppBar-colorDefault-8"
  >
    <div
      className="MuiToolbar-root-38 MuiToolbar-regular-40 MuiToolbar-gutters-39"
    >
      <p
        style={
          Object {
            "float": "right",
          }
        }
      >
        Â©2019 YoYo, Inc. All Rights Reserved.
      </p>
    </div>
  </header>
</div>
`;
